src/mad_spark_alt/core/prompt_classifier.py

```
@@ -3,13 +3,25 @@
 This module automatically detects question types and complexity to enable
 adaptive prompt selection for optimal QADI analysis results.
+
+DEPRECATED: This module is deprecated as of v1.x. Use SimpleQADIOrchestrator instead,
+which provides universal prompts without classification. This module will be removed in v2.0.0.
 """
 
 import re
+import warnings
 from dataclasses import dataclass
 from enum import Enum
 from typing import Any, Dict, List, Optional, Set, Tuple, cast
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import

(F401)